docs/sources/faq.md

```diff
@@ -234,46 +217,48 @@ When building Docker images on Debian and Ubuntu you may have seen errors like:
     unable to initialize frontend: Dialog
 
 These errors don't stop the image from being built but inform you that the
-installation process tried to open a dialog box, but was unable to. 
-Generally, these errors are safe to ignore.
+installation process tried to open a dialog box, but was unable to. Generally,
+these errors are safe to ignore.
 
-Some people circumvent these errors by changing the `DEBIAN_FRONTEND` 
+Some people circumvent these errors by changing the `DEBIAN_FRONTEND`
 environment variable inside the Dockerfile using:
 
     ENV DEBIAN_FRONTEND=noninteractive
 
-This prevents the installer from opening dialog boxes during installation 
-which stops the errors.
+This prevents the installer from opening dialog boxes during installation which
+stops the errors.
 
-While this may sound like a good idea, it *may* have side effects. 
-The `DEBIAN_FRONTEND` environment variable will be inherited by all 
-images and containers built from your image, effectively changing
-their behavior. People using those images will run into problems when
-installing software interactively, because installers will not show
-any dialog boxes.
+While this may sound like a good idea, it *may* have side effects. The
+`DEBIAN_FRONTEND` environment variable will be inherited by all images and
+containers built from your image, effectively changing their behavior. People
+using those images will run into problems when installing software
+interactively, because installers will not show any dialog boxes.
 
 Because of this, and because setting `DEBIAN_FRONTEND` to `noninteractive` is
 mainly a 'cosmetic' change, we *discourage* changing it.
 
-If you *really* need to change its setting, make sure to change it
-back to its [default value](https://www.debian.org/releases/stable/i386/ch05s03.html.en) 
+If you *really* need to change its setting, make sure to change it back to its
+[default value](https://www.debian.org/releases/stable/i386/ch05s03.html.en)
 afterwards.
 
```

Comment:
Oooh, I didn't realise @moxiegirl fixed the wrapping for the *whole* document because of my comment!

@moxiegirl my note on the `80-chars` wrapping was meant just for the *new* lines (but, I didn't yet check if the rest of the document also wrapped at 80-chars);

The general "rule" is to have only "related" changes in a pull-request, so that it's easier to track-down and (if needed) revert just *those* changes; in this case, have the new FAQ (wrapped to 80-chars) in *this* PR. And, (if you want to fix the 80-chars wrapping for the rest of the document), do that in a follow-up PR.

Given that this document doesn't change very frequently, it may not be a *huge* issue (but I'll leave that to the maintainers).

Feel free to blame me if I brought you into trouble; ["stroopwafels"](http://en.m.wikipedia.org/wiki/Stroopwafel) are on me for everyone, next time you visit the Netherlands. 😃